crypto/include/internal/asn1_int.h

```diff
@@ -63,6 +63,10 @@ struct evp_pkey_asn1_method_st {
     int (*set_pub_key) (EVP_PKEY *pk, const unsigned char *pub, size_t len);
     int (*get_priv_key) (const EVP_PKEY *pk, unsigned char *priv, size_t *len);
     int (*get_pub_key) (const EVP_PKEY *pk, unsigned char *pub, size_t *len);
+
+    /* Exports to providers */
+    int (*is_dirty) (const EVP_PKEY *pk);
```

Comment:
I think we'll need something to reset the dirty flag

Reply:
I was thinking that it'd be a job for `export_to`, but I do see the point with making it possible to clear from the exterior.  Will do

Reply:
Done